tests/integration_tests/fixtures/birth_names_dashboard.py

```diff
@@ -37,34 +37,32 @@
 def load_birth_names_data(
     birth_names_table_factory: Callable[[], Table], data_loader: DataLoader
 ):
-    birth_names_table: Table = birth_names_table_factory()
-    data_loader.load_table(birth_names_table)
-    yield
-    data_loader.remove_table(birth_names_table.table_name)
+    with app.app_context():
```

Comment:
Due to inheritance of fixtures only the outer fixture requires instantiating the app context.